libs/utils/utils.js

```diff
@@ -763,9 +763,9 @@ async function decorateIcons(area, config) {
   if (icons.length === 0) return;
   const { base } = config;
   loadStyle(`${base}/features/icons/icons.css`);
-  loadLink(`${base}/img/icons/icons.svg`, { rel: 'preload', as: 'fetch', crossorigin: 'anonymous' });
+  loadLink(`${base}/utils/federated.js`, { rel: 'preload', as: 'script', crossorigin: 'anonymous' });
   const { default: loadIcons } = await import('../features/icons/icons.js');
-  await loadIcons(icons, config);
+  await loadIcons(icons);
```

Comment:
I would have two suggestions two help the performance concerns on this:

- Preload any icon that's used in LCP//the first section (in utils.js)
- Load icons on a section to section basis rather than all within the document at once as @overmyheadandbody suggested as well.

Reply:
@mokimo Thanks for the suggestions. 
I've implemented the first one, preloading icons in 1st section, which does help a hair, I'm still doing some testing, just and update, but looking at the second one. it is loading those per section, Im going to try loading out of view ones postLPC and will let you know. also, Instead of preloading 1st section, should we look at whats in viewport?